src/preact.d.ts

```diff
@@ -361,390 +426,413 @@ declare namespace JSX {
 
-	interface HTMLAttributes extends preact.PreactHTMLAttributes, DOMAttributes {
+	interface HTMLAttributes extends PreactHTMLAttributes, DOMAttributes {
 		// Standard HTML Attributes
-		accept?:string;
-		acceptCharset?:string;
-		accessKey?:string;
-		action?:string;
-		allowFullScreen?:boolean;
-		allowTransparency?:boolean;
-		alt?:string;
-		async?:boolean;
-		autocomplete?:string;
-		autofocus?:boolean;
-		autoPlay?:boolean;
-		capture?:boolean;
-		cellPadding?:number | string;
-		cellSpacing?:number | string;
-		charSet?:string;
-		challenge?:string;
-		checked?:boolean;
-		class?:string | { [key:string]: boolean };
-		className?:string | { [key:string]: boolean };
-		cols?:number;
-		colSpan?:number;
-		content?:string;
-		contentEditable?:boolean;
-		contextMenu?:string;
-		controls?:boolean;
-		coords?:string;
-		crossOrigin?:string;
-		data?:string;
-		dateTime?:string;
-		default?:boolean;
-		defer?:boolean;
-		dir?:string;
-		disabled?:boolean;
-		download?:any;
-		draggable?:boolean;
-		encType?:string;
-		form?:string;
-		formAction?:string;
-		formEncType?:string;
-		formMethod?:string;
-		formNoValidate?:boolean;
-		formTarget?:string;
-		frameBorder?:number | string;
-		headers?:string;
-		height?:number | string;
-		hidden?:boolean;
-		high?:number;
-		href?:string;
-		hrefLang?:string;
-		for?:string;
-		httpEquiv?:string;
-		icon?:string;
-		id?:string;
-		inputMode?:string;
-		integrity?:string;
-		is?:string;
-		keyParams?:string;
-		keyType?:string;
-		kind?:string;
-		label?:string;
-		lang?:string;
-		list?:string;
-		loop?:boolean;
-		low?:number;
-		manifest?:string;
-		marginHeight?:number;
-		marginWidth?:number;
-		max?:number | string;
-		maxLength?:number;
-		media?:string;
-		mediaGroup?:string;
-		method?:string;
-		min?:number | string;
-		minLength?:number;
-		multiple?:boolean;
-		muted?:boolean;
-		name?:string;
-		noValidate?:boolean;
-		open?:boolean;
-		optimum?:number;
-		pattern?:string;
-		placeholder?:string;
-		poster?:string;
-		preload?:string;
-		radioGroup?:string;
-		readOnly?:boolean;
-		rel?:string;
-		required?:boolean;
-		role?:string;
-		rows?:number;
-		rowSpan?:number;
-		sandbox?:string;
-		scope?:string;
-		scoped?:boolean;
-		scrolling?:string;
-		seamless?:boolean;
-		selected?:boolean;
-		shape?:string;
-		size?:number;
-		sizes?:string;
-		slot?:string;
-		span?:number;
-		spellCheck?:boolean;
-		src?:string;
-		srcset?:string;
-		srcDoc?:string;
-		srcLang?:string;
-		srcSet?:string;
-		start?:number;
-		step?:number | string;
-		style?:any;
-		summary?:string;
-		tabIndex?:number;
-		target?:string;
-		title?:string;
-		type?:string;
-		useMap?:string;
-		value?:string | string[];
-		width?:number | string;
-		wmode?:string;
-		wrap?:string;
+		accept?: string;
+		acceptCharset?: string;
+		accessKey?: string;
+		action?: string;
+		allowFullScreen?: boolean;
+		allowTransparency?: boolean;
+		alt?: string;
+		async?: boolean;
+		autocomplete?: string;
+		autofocus?: boolean;
```

Comment:
I've got a separate PR to make this change, so maybe we can just merge that in after this... _But_, if you want, I believe `autoComplete` and `autoFocus` should be camel-cased (as I just wrote them).